docs/src/app_dev/app_val_tokenomics/cc_xfer_nomics.rst

```diff
@@ -0,0 +1,82 @@
+..
+   Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+..
+   SPDX-License-Identifier: Apache-2.0
+
+.. _cc_xfer_nomics:
+
+Canton Coin Transfer
+====================
+
+A CC ``Amulet`` `Transfer <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-AmuletRules.html#type-splice-amuletrules-transfer-72721>`__
+or `TwoStepTransfer <https://docs.dev.sync.global/app_dev/api/splice-amulet/Splice-Amulet-TwoStepTransfer.html#splice-amulet-twosteptransfer>`__
```

Comment:
A `TwoStepTransfer` is an amulet-internal implementation contract, which is built on top of `AmuletRules_Transfer`, which is the one and only choice that actually implements a transfer. So let's not throw these two into the same bucket.

I know you've asked me not to review the structure, but here I'd like to push back on this. I'd really appreciate this explanation here focusing on `AmuletRules_Transfer` and have that explain well how to go from its arguments to the activity being recorded.

Once that's done you can explain the various workflows that build on `AmuletRules_Transfer` and how they set the `provider` and the `beneficiaries`.

Reply:
Done

Reply:
Done